gcc_8/test.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
-sudo docker run -t -d --name conangcc8 lasote/conangcc8 && \
-sudo docker exec conangcc8 sudo pip install -U conan_package_tools && \
-sudo docker exec conangcc8 sudo pip install -U conan && \
-sudo docker exec conangcc8 conan user && \
-sudo docker exec conangcc8 conan install gtest/1.8.0@lasote/stable -s arch=x86_64 -s compiler=gcc -s compiler.version=7 -s compiler.libcxx=libstdc++ --build && \
-sudo docker exec conangcc8 conan install gtest/1.8.0@lasote/stable -s arch=x86 -s compiler=gcc -s compiler.version=7 -s compiler.libcxx=libstdc++ --build && \
-sudo docker stop conangcc8 && \
-sudo docker rm conangcc8
+sudo docker run -t -d --name gcc8 conanio/gcc8 && \
+sudo docker exec gcc8 sudo pip install -U conan_package_tools && \
+sudo docker exec gcc8 sudo pip install -U conan && \
+sudo docker exec gcc8 conan user && \
+sudo docker exec gcc8 conan install gtest/1.8.0@conanio/stable -s arch=x86_64 -s compiler=gcc -s compiler.version=7 -s compiler.libcxx=libstdc++ --build && \
+sudo docker exec gcc8 conan install gtest/1.8.0@conanio/stable -s arch=x86 -s compiler=gcc -s compiler.version=7 -s compiler.libcxx=libstdc++ --build && \
```

Comment:
I guess this should be `gtest/1.8.1@bincrafters/stable` instead in all test.sh files

Reply:
Evil find and replace